driver/annotations.md

```diff
@@ -0,0 +1,224 @@
+# Adding UAST Annotations
+
+# Intro
+
+Once you have written the [native part of a programming language's
+driver](driver/sdk.md) the
+next step is to write the annotation code. This is the Go language code that will
+establish the rules to transform the native driver AST into the UAST and the
+implementation uses two objects, the ObjectToNoder and the AnnotationRules on two
+separate files.  Most of the files related to the Go part of the driver are
+auto-generated by the [`bblfsh-sdk init`
+tool](driver/sdk.md), but in
+order to translate the native AST to the UAST you need to complete two skeleton
+files that are referenced in the (auto-generated) driver/main.go file.
+
+## File: parser.go
+
+### ObjectToNoder struct
+
+This is a struct that you'll need to fill in the `driver/normalizer/parser.go`
+file. It serves to establish the role of some internal keys in the native AST into
+specific key roles in the UAST. This is needed so the rule applying process knows
+what parts of the native AST use to do the translation. The skeleton of this
+struct will be automatically generated by the `bblfsh-sdk init` command. 
+
+The meaning of the fields in the ObjectToNoder struct are:
+
+- InternalTypeKey (string): this is the name of the key that the native AST uses
```

Comment:
Mmmm I think that the more concise descriptions are better for an API doc as they're and these (more developed) are better on the book. No?